repldex/backend/templates/entry.html

```diff
@@ -8,87 +8,60 @@
+            popupContainerEl.classList.add('popupContainer')
+            popupInnerEl.classList.add('popupInner')
+            popupTitleEl.classList.add('popupTitle')
+            popupContentEl.classList.add('popupContent')
+            popupFaviconEl.classList.add('popupFavicon')
+            anchor.popupContainerEl = popupContainerEl
 
-	popupInnerEl.appendChild(popupTitleEl)
-	popupContainerEl.appendChild(popupInnerEl)
+            popupInnerEl.appendChild(popupTitleEl)
+            popupContainerEl.appendChild(popupInnerEl)
 
-	anchor.parentElement.insertBefore(popupContainerEl, anchor)
+            anchor.parentElement.insertBefore(popupContainerEl, anchor)
 
-	popupContainerEl.style.display = 'none'
+            popupContainerEl.style.display = 'none'
 
 
-	fetch(`/api/website-title?url=${anchor.href}`).then(async(r) => {
-		resp = await r.json()
-		popupTitleEl.innerText = resp.title
+            fetch(`/api/website-title?url=${anchor.href}`).then(async(r) => {
+                resp = await r.json()
+                popupTitleEl.innerText = resp.title
 
-		if (resp.favicon) {
-			popupInnerEl.appendChild(popupFaviconEl)
-			popupFaviconEl.src = resp.favicon
-		}
-		if (resp.content) {
-			popupInnerEl.appendChild(popupContentEl)
-			popupContentEl.innerText = resp.content.substr(0, 200)
-		}
-		popupContainerEl.style.display = 'inline-block'
+                if (resp.favicon) {
+                    popupInnerEl.appendChild(popupFaviconEl)
+                    popupFaviconEl.src = resp.favicon
+                }
+                if (resp.content) {
+                    popupInnerEl.appendChild(popupContentEl)
+                    popupContentEl.innerText = resp.content.substr(0, 200)
+                }
+                popupContainerEl.style.display = 'inline-block'
 
 
-	})
+            })
 
 
+        })
+        anchor.addEventListener('mouseout', () => {
+            anchor.popupContainerEl.remove()
+            anchor.popupContainerEl = undefined
+        })
+    }
 })
-anchor.addEventListener('mouseout', () => {
-	anchor.popupContainerEl.remove()
-	anchor.popupContainerEl = undefined
-})
-}
-	
-})
-
-var idkwhyineedthis=true; //dont even ask
-
-function tts(){
-	var msg=new SpeechSynthesisUtterance();
-	var voices=speechSynthesis.getVoices();
-	var voice=document.getElementById("voice").value;
-	msg.voice=voices[voice];
-	msg.text="{{nohtml_content}}";
-	if(idkwhyineedthis){idkwhyineedthis=false;return;}
-	speechSynthesis.speak(msg);
-}
-
-function translate(){
-  var language=document.getElementById("language").value;
-  let language = {1:"english", 3:"pirate", 5: "shakespeare"};
-  <option value="1" selected>English</option>
-		<option value="2">Lolcat</option>
-		<option value="3">Pirate</option>
-    <option value="4">Morse Code</option>
-    <option value="5">Shakespeare</option>
-    <option value="6">Gungan</option>
-    <option value="7">Leet</option>
-    <option value="8">Pig Latin</option>
-}
-
```

Comment:
Removal of TTS and translations should be in a different PR

Reply:
It corresponds to a Lighthouse issue, so idrc

Reply:
yeah seems fair